lib/spack/docs/getting_started.rst

```diff
@@ -885,52 +894,73 @@ Environment Modules
 In order to use Spack's generated environment modules, you must have
 installed one of *Environment Modules* or *Lmod*.  On many Linux
 distributions, this can be installed from the vendor's repository.  For
-example: ``yum install environment-modules`` (Fedora/RHEL/CentOS).  If
-your Linux distribution does not have Environment Modules, you can get it
-with Spack:
+example: ``yum install environment-modules`` (Fedora/RHEL/CentOS). If
+your Linux distribution does not have Environment Modules, Spack can
+build it for you!
 
-#. Consider using system tcl (as long as your system has Tcl version 8.0 or later):
+What follows are three steps describing how to install and use environment-modules with spack.
 
-   #) Identify its location using ``which tclsh``
-   #) Identify its version using ``echo 'puts $tcl_version;exit 0' | tclsh``
-   #) Add to ``~/.spack/packages.yaml`` and modify as appropriate:
+#. Install ``environment-modules``.
 
-      .. code-block:: yaml
+   * Spack can build and install ``environment-modules`` for you.
```

Comment:
IMO this wording is slightly awkward since spack is building `environment-modules` for you in both cases. Perhaps:

> `spack bootstrap` will build environment-modules for you (and may build other packages that are useful to the operation of Spack).

Other than that LGTM - make that change and I'll merge this.